docs/guide/basic-usage.md

```diff
@@ -30,7 +30,7 @@ Testing
 -----------
 
 Define a `tests` alias in your console config. For example, for the `basic` project template, this should be added
-to the `console.php` configuration: `Yii::setAlias('tests', __DIR__ . '/../tests');`
+to the `console.php` configuration: `Yii::setAlias('tests', __DIR__ . '/tests/codeception');`
```

Comment:
if this code goes into [console.php](https://github.com/yiisoft/yii2-app-basic/blob/a2b4a8677c27d7a762d4432c237f2034a3b1b4ba/config/console.php) is should go one directory upwards. Should be `/../tests/codeception`